Source/ApiForm.swift

```diff
@@ -1,80 +1,105 @@
 import RealmSwift
 import Alamofire
 
-public enum ApiModelStatus {
-    case None
-    case Successful(Int)
-    case Unauthorized(Int)
-    case Invalid(Int)
-    case ServerError(Int)
-    
-    init(statusCode: Int) {
-        if statusCode >= 200 && statusCode <= 299 {
-            self = .Successful(statusCode)
-        } else if statusCode == 401 {
-            self = .Unauthorized(statusCode)
-        } else if statusCode >= 400 && statusCode <= 499 {
-            self = .Invalid(statusCode)
-        } else if statusCode >= 500 && statusCode <= 599 {
-            self = .ServerError(statusCode)
-        } else {
-            self = .None
+public protocol ApiModelResponseable {
```

Comment:
If that's the case then I don't really see the need for it. For me at least it just causes confusion in the source code, as I can't really see a reason you might inherit from this. I might be wrong though, what scenarios did you have in mind?